content/docs/fundamentals/converting/resolving-errors@zh_CN.md

```diff
@@ -0,0 +1,350 @@
+---
+$title: 解决验证错误
+$order: 2
+toc: true
+---
+
+[TOC]
+
+在本部分中，我们将讲解 AMP 网页中的 AMP 验证错误及其解决方法。请注意，这些错误可能会在您的控制台中以不同的顺序出现。
+
+## 包含字符集
+
+首先，我们将修正以下错误：
+
+<pre class="error-text">
```

Comment:
All of these code blocks need to be in EN, so please update with EN translation (https://www.ampproject.org/docs/fundamentals/converting/resolving-errors).